resources/views/members/index.blade.php

```diff
@@ -130,43 +135,43 @@
 						<td>{{ $member->telefoon }}</td>
 						<td><a href="mailto:{{ $member->email }}">{{ $member->email }}</a></td>
 					</tr>
-				@endforeach
-			</tbody>
-		</table>
+					@endforeach
+				</tbody>
+			</table>
+		</div>
+		@endcan
 	</div>
-	@endcan
-  </div>
 
 </div>
 @endsection
 
 @section('footer')
 <!-- These scripts load DataTables -->
 <script type="text/javascript">
-$( document ).ready(function() {
+	$( document ).ready(function() {
+
+	// Create the column options for the members table (number of columns can vary with user capabilities)
+	let memberColumnCount = $('#currentMembersTable').find('tr:first th').length;
+	let orderableColumnIndices = [0, 2, 3];
+	let columnOptions = [];
+	for (let i = 0; i < memberColumnCount; i++) {
+		let option = (orderableColumnIndices.includes(i)) ? null : {'orderable': false};
```

Comment:
```suggestion
		let option = (orderableColumnIndices.indexOf(i) > -1) ? null : {'orderable': false};
```
`Array.prototype.includes` is niet heeel goed ondersteund over browsers heen, gebruik `indexOf() > -1`.